stdlib/knative/src/main/ballerina/src/knative/annotation.bal

```diff
@@ -0,0 +1,250 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+# Knative annotation configuration.
+#
+#
+#
+# Metadata for artifacts
+#
+# + name - Name of the resource. Default is `"<OUPUT_FILE_NAME>-<RESOURCE_TYPE>"`.
+# + labels - Map of labels for the resource.
+# + annotations - Map of annotations for resource.
```

Comment:
In some type documentations, filed descriptions are ended with a fullstop while in others are not. Best practice is not to use fullstop for field descriptions. (Only add fulllstop for record descriptions). Check all the places and fix.

Reply:
checked and added fullstops.

Reply:
It seems the fix is incorrect. We need to remove fullstops from the field descriptions.

Reply:
changes added, please check